src/main/java/org/reservationapplication/repository/CoworkingSpaceRepository.java

```diff
@@ -43,14 +44,14 @@ public void save(List<CoworkingSpace> coworkingSpaces) {
     @Override
     public List<CoworkingSpace> read() {
         File file = new File(COWORKING_FILE_NAME);
-        List<CoworkingSpace> generalCoworkingSpace;
+        List<CoworkingSpace> allCoworkingSpaces;
 
         try {
             if (file.exists() && file.length() > 0) {
-                generalCoworkingSpace = objectMapper.readValue(file, new TypeReference<>(){});
-                updateID(generalCoworkingSpace);
+                allCoworkingSpaces = objectMapper.readValue(file, new TypeReference<>(){});
```

Comment:
would be great it you made this part abstract, common for both Repositories. 
Create interface with load and save methods (name smth like DataStorage, without Json and Workspace words).  Create abstract class JsonDataStorage (meaning that we will work with Json in it) and afret that create WorkspaceDataStorage where you just define type and file name